packages/ui/src/components/AccessibilityGuide.md

```diff
@@ -0,0 +1,78 @@
+# 1. Accessibility guide
+
+We want to provide following accessibility features:
+- roles & aria attributes,
+- keyboard accessibility,
+- proper typography, colors and contrast ratio.
+
+**Source of truth:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
```

Comment:
As additional URL I'll suggest to use the W3C standarts: https://www.w3.org/standards/webdesign/accessibility